x-pack/plugins/observability/public/hooks/slo/use_fetch_active_alerts.ts

```diff
@@ -82,7 +82,7 @@ export function useFetchActiveAlerts({
                   bool: {
                     filter: [
                       { term: { 'slo.id': sloId } },
-                      { term: { 'slo.instanceId': instanceId } },
+                      ...(instanceId === '*' ? [] : [{ term: { 'slo.instanceId': instanceId } }]),
```

Comment:
I'll try it locally as well, but if I understand correctly if instanceId is not `*` query changes to?

```
[
  { term: { 'slo.id': sloId } },
  [{ term: { 'slo.instanceId': instanceId } }]
]
```

This part is used in SLO list page and SLO detail page. Let's make sure it still works fine there. Do we have any tests for this?

One reminder, we use different query for calculating active alerts in SLO pages vs SLO embeddable. I will create a ticket to unify the query in both places. Linking this for reference https://github.com/elastic/kibana/issues/172448

Reply:
cc @kdelemme since you wrote this part, I am tagging you

Reply:
Technically, instanceId can be '*' and the alert stores the instanceId as `*` as well, so I'm not sure this filter adds something? but I need to try the query locally